src/components/molecules/OnlineStats/OnlineStats.tsx

```diff
@@ -44,7 +44,7 @@ const PotLuckButton: React.FC<PotLuckButtonProps> = ({
     afterSelect();
 
     // there is a bug in useConnectCurrentVenue that does not update correctly on url change
-    window.location.href = venueInsideUrl(randomVenue.id);
+    openUrl(venueInsideUrl(randomVenue.id));
```

Comment:
how does `venueInsideUrl` and the get room url function that `openUrl` uses work together?

Reply:
apparently I am thinking of `openRoomUrl`. Does that serve the purpose of `venueInsideUrl`?

Reply:
As below, we won't need `openRoomUrl` here because `venueInsideUrl` will always generate a URL prefixed with `http`.